tensorboard/plugins/audio/tf_audio_dashboard/tf-audio-loader.html

```diff
@@ -131,18 +147,22 @@
           type: Boolean,
           computed: "_computeHasMultipleSteps(_steps)",
         },
-        _stepValue: {
-          type: Number,
-          computed: "_computeStepValue(_stepIndex)",
-        },
         _currentDatum: {
-          type: Object,  // {wall_time, step, content_type, url}
+          type: Object,  // {wall_time, step, label, url}
           computed: "_computeCurrentDatum(_steps, _stepIndex)",
         },
         _maxStepIndex: {
           type: Number,
           computed: "_computeMaxStepIndex(_steps)",
         },
+        _sampleText: {
+          type: String,
+          computed: "_computeSampleText(sample)",
+        },
+        _hasMultipleSamples: {
+          type: Boolean,
+          computed: "_computeHasMultipleSamples(ofSamples)",
```

Comment:
(I'm using `totalSamples` because with `numberOfSamples` one is more likely to incorrectly infer that a particular card has multiple samples; using `totalSamples` clarifies that the aggregation is done at a higher level than the individual cards.)